gensim/api/__init__.py

```diff
@@ -1,34 +1,41 @@
-import subprocess
+from __future__ import print_function
 import json
 import sys
 import os
-import pip
-import importlib
-from pathlib import Path
-import os
+import six
+import tarfile
+try:
+    import urllib.request as urllib
+except ImportError:
+    import urllib
+
 try:
     from urllib.request import urlopen
 except ImportError:
     from urllib2 import urlopen
 
 
 def download(file):
-    url = "https://github.com/chaitaliSaini/Corpus_and_models/releases/download/"
-    data = catalogue(False)
-    corpuses = data['gensim']['corpus']
-    models = data['gensim']['model']
-    if file not in corpuses and file not in models:
-        print("Incorrect corpus/model name.")
-    else:
-        url = url+file+"/"+file+".tar.gz"
-        print("Downloading {m}".format(m=file))
-        subprocess.call(
-            [sys.executable, '-m', 'pip', 'install', '--no-cache-dir', url],
-            env=os.environ.copy())
+    url = "https://github.com/chaitaliSaini/Corpus_and_models/releases/"
+    url = url+"download/"+file+"/"+file+".tar.gz"
+    user_dir = os.path.expanduser('~')
+    base_dir = os.path.join(user_dir, 'gensim-data')
+    extracted_folder_dir = os.path.join(base_dir, file)
+    if not os.path.exists(base_dir):
+        os.makedirs(base_dir)
```

Comment:
I just read about it and logging is way better than using print. So i'll add logging.